Resources/Views/internal-linking.leaf

```diff
@@ -1,7 +1,7 @@
 <div class="internal-linking">
   <div class="internal-linking__container">
     <div class="internal-linking__title">
-      <img width="12" height="16" src="img/lightning.svg" />
+      <img width="12" height="16" src="img/lightning.svg" alt="lightning" />
```

Comment:
Do we need this alt? That image is presentational, I think we should leave `alt=""` here.

Reply:
Empty alt tag is the way to go here. ☝️